### PR TITLE
Testing RedBlackTreeNode with TravisCI

### PR DESCRIPTION
**Pull request by HarsheetKakar**
#### References to other Issues or PRs or Relevant literature

Draft PR to test with TravisCI

### CODE REVIEW

File: pydatastructs/utils/misc_util.py [hunk 1]
```diff
@@ -35,6 +36,41 @@ class TreeNode(Node):
...
+        obj.is_root = False
+        return obj
+
+    def __str__(self):
+        """
+        Used for printing.
+        """
+        return str((self.left, self.key, self.data, self.right))
+
+class RedBlackTreeNode(Node):
```
Comment (czgdp1807) [inline pydatastructs/utils/misc_util.py]: `Node` -> `TreeNode`?

Comment (HarsheetKakar) [reply pydatastructs/utils/misc_util.py]: No luck, still giving the same error.

Comment (HarsheetKakar) [reply pydatastructs/utils/misc_util.py]: error is originating from `OneDimensionalArray.__setitem__()`

Comment (HarsheetKakar) [reply pydatastructs/utils/misc_util.py]: I found the bug. will send the PR with its possible fix.

**Pull request closed without merging**